### PR TITLE
checkin .travis.yml for travis-ci

### PR DESCRIPTION
**Pull request by cameronphchen**
testing for linux works. But there's a problem with osx due to the clang-omp removal from brew #84 . (can't brew install clang-omp on travis-ci server). Issue #63

### CODE REVIEW

**Comment by buildbot-princeton** [comment]
Can one of the admins verify this patch?

**Comment by buildbot-princeton** [comment]
Can one of the admins verify this patch?

**Comment by mihaic** [comment]
Add to whitelist.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+
+    - os: osx
+      osx_image: xcode7.3
+      sudo: required
+      language: generic
+      env: HINT=ElCapitan MPI=openmpi
+      before_install:
+          - brew update
+          - brew install clang-omp
```
Comment (mihaic) [inline .travis.yml]: Could you please change to `llvm` and change the env vars according to PR #82? This should make it work, so you should also remove the `allow_failures` and the `gcc` attempted workaround.

**Comment by cameronphchen** [comment]
@mihaic , thanks for the comments. I have updated the pull request according to it.

**Comment by mihaic** [comment]
Thanks, @cameronphchen. We can merge this. We need you to update your branch so it is up-to-date with master first. It should be as easy as pressing the "Update branch" button on this page. (I cannot do it because I do not have write access to your fork.)

**Comment by cameronphchen** [comment]
@mihaic Just did. Thanks!

**Pull request merged**